solr/core/src/java/org/apache/solr/response/JacksonJsonWriter.java

```diff
@@ -29,6 +29,16 @@
 import org.apache.solr.request.SolrQueryRequest;
 
 public class JacksonJsonWriter extends BinaryResponseWriter {
+
+  protected final JsonFactory jsonfactory;
```

Comment:
`protected static final JsonFactory jsonfactory = new JsonFactory();`
    I'm sorry, maybe it would be better to create a factory right here than in the constructor (just like you did with "PrettyPrinter pretty = ")? Then the "public JacksonJsonWriter(){super()..." constructor itself can be deleted below, since it will no longer be needed.